akka-testkit/src/main/scala/akka/testkit/SocketUtil.scala

```diff
@@ -24,7 +24,7 @@ object SocketUtil {
       SocketUtil.temporaryServerAddress(address = "127.20.0.0")
       true
     } catch {
-      case e: java.net.BindException ⇒
+      case _: java.net.BindException ⇒
         false
     }
   }
```

Comment:
Out of scope for your change, but seems cleaner
```suggestion
private val canBindOnAlternativeLoopbackAddresses: Boolean =
  Try(SocketUtil.temporaryServerAddress(address = "127.20.0.0"))
    .recover { case _: java.net.BindException ⇒ false}.isSuccess
```